webapp/store/models.py

```diff
@@ -33,11 +35,12 @@ def search_entities(
         entities = cs.search(
             query,
             doc_type=entity_type,
-            tags=tags,
-            sort=sort,
-            series=series,
             includes=includes,
+            limit=SEARCH_LIMIT,
```

Comment:
Is this a driveby fix for the 10 limit in search results?

Reply:
It is!